src/ProcessManager/ChromeManager.php

```diff
@@ -30,13 +30,26 @@ final class ChromeManager implements BrowserManagerInterface
     private $arguments;
     private $options;
 
+    private $experimentalOptions = [];
+    private $extensions = [];
+
     public function __construct(?string $chromeDriverBinary = null, ?array $arguments = null, array $options = [])
     {
         $this->options = array_merge($this->getDefaultOptions(), $options);
         $this->process = new Process([$chromeDriverBinary ?: $this->findChromeDriverBinary(), '--port='.$this->options['port']], null, null, null, null);
         $this->arguments = $arguments ?? $this->getDefaultArguments();
     }
 
+    public function setExtensions(array $extensions)
```

Comment:
The `$extensions` argument should be an array of strings with each string being a path to an extension.

This need is not clear from the method name alone. Could you document the argument needs?